src/gui/widgets/SideBarWidget.cpp

```diff
@@ -39,6 +39,10 @@ SideBarWidget::SideBarWidget( const QString & _title, const QPixmap & _icon,
 	m_layout = new QVBoxLayout( m_contents );
 	m_layout->setSpacing( 5 );
 	m_layout->setMargin( 0 );
+	m_closeBtn = new QToolButton(this);
+	m_closeBtn->setText(QString("X"));
```

Comment:
An `X`-shaped icon will work better than a literal `X`. See this example(`src/gui/SubWindow.cpp`):
https://github.com/LMMS/lmms/blob/e1ef5fad0d5d8ce9b74b0c970c128cb90d695783/src/gui/SubWindow.cpp#L56-L57

Reply:
Ah! I was curious where to get that from. I couldn't find it for some reason. I'll fix that up.